### ISSUE TITLE
MAC OS Big Sur - problematic

### ISSUE THREAD

**Issue by Rybeast** [Admiral-Fish/RaidFinder#73]
Unable to generate frames using RaidFinder on Mac OS - settings and seed verified by two others.
Filters for IVs do not show numbers clearly - boxes are not expanding to allow you to see any number.

**Comment by Admiral-Fish** [comment]
`Unable to generate frames using RaidFinder on Mac OS - settings and seed verified by two others.`

I have no clue what you mean. Provide more information.

`Filters for IVs do not show numbers clearly - boxes are not expanding to allow you to see any number.`

Have you tried resizing the window?

**Comment by Rybeast** [comment]
I am unable to generate anything to identify how many frames away particular Pokemon in the den are. All the drop down boxes and filters have been correctly used along with the filters, but I am unable to generate anything when I click 'generate'. Two others have looked at my setup and confirmed that is correct, and have been able to generate information to identify how many frames away particular Pokemon are.

When you say resizing the window - I have maximised the window, minimised the window, shut down and reloaded, but the IVs still stay 'hidden'. I've tried to drag and resize that section, but to no avail.
